ts/Stock/StockTools/StockToolbar.ts

```diff
@@ -649,6 +668,20 @@ class Toolbar {
         }
     }
 
+    /**
+     * Remove highcharts-active class from button.
+     * @private
+     */
+    public removeButtonActiveClass(
```

Comment:
I'm afraid that 4fd095a6fa7a99d7c279fe748321becbb437430f removed the usage of this method, and we just have a function, but it's not used anywhere.

So we either need to remove this function, or revert the changes from the commit that removed the usage.